frontend/public/locales/fr/apply.json

```diff
@@ -71,12 +71,30 @@
   },
   "confirm": {
     "page-title": "(FR) Confirmation",
-    "p-one": "(FR) You can use the CDCP Status Checker or call 1-833-537-4342 to verify the status of your application. Your status should be available within the next 48 hours.",
-    "p-two": "(FR) If you haven't already, sign up for a <MSCALink>My Service Canada Account (MSCA)</MSCALink> — a secure online portal designed to streamline your application management process.",
-    "p-three": "(FR) Once your application is processed, you will be able to sign in to MSCA to view/update your personal information and view any letters sent to you by the Government of Canada regarding the CDCP.",
-    "p-four": "(FR) If you are deemed eligible, Sun Life will send you a welcome package by mail. Your welcome package will include your member card and details about the CDCP, including your coverage start date. The CDCP will not reimburse you for dental services received before that date.",
-    "p-five": "(FR) If you are deemed ineligible and circumstances change that may impact your eligibility for the CDCP, you can submit a new application.",
-    "p-six": "(FR) For more information, please contact <dentalContactUsLink>Service Canada.</dentalContactUsLink>",
+    "keep-copy": "(FR) Keep a copy of your application",
+    "print-copy-one": "(FR) You will",
+    "not-bold": "not",
+    "print-copy-two": "be receiving a",
+    "confirm-email-bold": "confirmation email",
+    "print-copy-three": ". It is important to keep a copy of your application. You can use this button to print.",
```

Comment:
Consider using the `Trans` component instead of breaking this sentence down into multiple parts.

See https://react.i18next.com/latest/trans-component